Update collaboration - TO REVIEW.md (#188)
SenseNet/docs.sensenet.com: content/concepts/collaboration.md

metaTitle: "sensenet - Versioning"
metaDescription: "sensenet Versioning"
---

In many cases developers and non-developers have to work together during a project. sensenet enables collaborative content management, keeping all your content on a single hub.

# Collaboration features

> In sensenet it is possible to make real-time editing together with your collaborators, thanks to **Office Online Editing**.  **Versioning** feature is the management of changes to documents and other information stored in a file system or repository.

Reviewer: ```suggestion
In sensenet it is possible to make real-time editing together with your collaborators, thanks to **Office Online Editing**. 
**Versioning** let's you track changes of documents and other information stored in the repository.
```

> Using **simple approval**, contents needs to be sent for approval and can only be published when the review is finished and modification requests has been adapted.

Reviewer: ```suggestion
With **simple approval** feature, contents needs to be sent for approval and can only be published when the review is finished and modification requests has been adapted.
```

> Created around the concept “everything is content”, in sensenet it is easy to **share** not only a single content (like a document) but a whole folder or a workspace with other users.

Reviewer: ```suggestion
sensenet let's you easily **share** not only a single content (like a document) but whole folders or workspace with other users.
```
